### PR TITLE
[4.0]Add missing trait 

### PR DESCRIPTION
**Pull request by rdeutz**
Pull Request for Issue #20448

### Summary of Changes
Just added the trait as other classes do

### Testing Instructions
see #20448

### CODE REVIEW

**Comment by infograf768** [comment]
This does not break #20448 . I would not know how to test it specifically.

File: libraries/src/Application/WebApplication.php [hunk 1]
```diff
@@ -31,7 +32,7 @@
  */
 abstract class WebApplication extends AbstractWebApplication implements DispatcherAwareInterface
 {
-	use DispatcherAwareTrait, EventAware, IdentityAware;
+	use DispatcherAwareTrait, EventAware, IdentityAware,ContainerAwareTrait;
```
Comment (Quy) [inline libraries/src/Application/WebApplication.php]: Add space after comma.

**Comment by laoneo** [comment]
But then we should remove the trait from the CMSApplication and pass the container to the web application in the constructor.

**Comment by rdeutz** [comment]
@Quy shouldn't phpcs have catch this?

**Comment by Quy** [comment]
I don't know. Let's check with @photodude

**Comment by photodude** [comment]
@Quy @rdeutz, I'm not sure if the PHPCS 1.5.x version currently used on the CMS would catch that missing space after the comma in the `use` list. I believe the PHPCS2.x and 3.x versions should, but I haven't run a specific test to verify. The PHPCS2.x and 3.x versions are much more accurate and are more likely to catch the code style errors. 

We need to finish cleaning up the existing CMS code style violations so we make the move to the PHPCS2.x and 3.x versions. (I've been busy and haven't had time to work on it recently)

**Pull request closed without merging**